silkworm/silkrpc/commands/debug_api.cpp

```diff
@@ -329,27 +321,46 @@ awaitable<void> DebugRpcApi::handle_debug_trace_call(const nlohmann::json& reque
     auto tx = co_await database_->begin();
 
     try {
-        ethdb::TransactionDatabase tx_database{*tx};
-        ethdb::kv::CachedDatabase cached_database{block_number_or_hash, *tx, *state_cache_};
+        const auto call = params[0].get<Call>();
```

Comment:
`const auto bundles = params[0].get<Bundles>();` within `DebugRpcApi::handle_debug_trace_call_many` below could also throw now. That too would need to be put into the try catch.

Reply:
fixed